From 8329154c6a32b316c1ad002ca577580af5004915 Mon Sep 17 00:00:00 2001
From: Ian Wahbe <ian@wahbe.com>
Date: Tue, 21 Mar 2023 14:05:41 +0100
Subject: [PATCH] fix module name

---
 go.mod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/go.mod b/go.mod
index b6a3db8..8d2b04a 100644
--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module github.com/terraform-providers/terraform-provider-rabbitmq
+module github.com/cyrilgdn/terraform-provider-rabbitmq
 
 require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.21.0
-- 
2.39.1

