From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ian Wahbe <ian@wahbe.com>
Date: Tue, 21 Mar 2023 14:05:41 +0100
Subject: [PATCH] fix module name


diff --git a/go.mod b/go.mod
index 75cf142..18f4109 100644
--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module github.com/terraform-providers/terraform-provider-rabbitmq
+module github.com/cyrilgdn/terraform-provider-rabbitmq
 
 require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.37.0
